sbi.adoc

```diff
@@ -1,3 +1,32 @@
 [[sbi]]
 == SBI Requirements
 
+The supervisor binary interface (SBI) cite: [SBI] specification defines the interface
+between the supervisor mode and the next higher privilege mode. This section
+defines the mandatory SBI version and extensions implemented by the higher
+privilege mode in order to be compatible with this specification.
+
+=== Minimum SBI version
+The SBI implementation must be v1.0 or higher. 
```

Comment:
The wording should explicitly relate to the specification version and not to an implementation version, e.g:

The SBI implementation must conform to the SBI v1.0 specification.

Reply:
"v1.0 or higher" would allow to conform to a future version even if it contradicts the requirements of v1.0. That is why I think we should relate to a specific version.

Reply:
sgtm.